xgboost.ipynb

```diff
@@ -0,0 +1,227 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%matplotlib inline"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import os\n",
+    "from dask import compute, persist\n",
+    "from dask.distributed import Client, progress\n",
+    "\n",
+    "client = Client(os.environ.get(\"DISTRIBUTED_ADDRESS\"))  # connect to cluster \n",
+    "client"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "First we create a bunch of synthetic data:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import dask.array as da\n",
+    "from dask_ml.model_selection import train_test_split\n",
+    "import numpy as np\n",
+    "\n",
+    "n, d = int(100e3), 20\n",
+    "da.random.seed(42)\n",
+    "X = da.random.normal(size=(n, d), chunks=n // 100)\n",
+    "w_star = da.random.uniform(size=d, chunks=d)\n",
+    "w_star = w_star**3\n",
+    "y = da.sign(X @ w_star)\n",
+    "\n",
+    "# for binary:logistic objective, only [0, 1] labels allowed\n",
+    "y = (y + 1) / 2"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Now let's separate into a training set and testing set, which will allow for good evaluation after training."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "X_train, X_test, y_train, y_test = train_test_split(X, y, test_size=0.15)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Now, let's try and predict from this data with [dask-xgboost][dxgb]:\n",
+    "\n",
+    "[dxgb]:https://github.com/dask/dask-xgboost"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import dask_xgboost as dxgb\n",
+    "import xgboost as xgb"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "dask-xgboost is a small wrapper around xgboost, and will behave the same as xgboost.\n",
+    "\n",
+    "During training it will take care of handling the data, and allow xgboost to use it's own distributed scheduler."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%%time\n",
+    "params = {'objective': 'binary:logistic', 'nround': 1000, \n",
+    "          'max_depth': 5, 'eta': 0.01, 'subsample': 0.5, \n",
+    "          'min_child_weight': 0.5}\n",
+    "\n",
+    "bst = dxgb.train(client, params, X_train, y_train)\n",
```

Comment:
It would be good to remove the need to pass in `client` here in dask-xgboost and instead use `dask.distributed.default_client` if client is not explicitly provided.  Can I ask you to submit a small PR to the dask-xgboost library with this fix?  

I'm not sure about the best way to deprecate this cleanly.  It might also be a good idea to open an issue to have a longer conversation with @TomAugspurger .